apps/api/src/app/workflows-v1-deprecated/workflowControllerDeprecated.ts

```diff
@@ -48,12 +48,16 @@ import { GetWorkflowVariablesCommand } from './usecases/get-workflow-variables/g
 import { UserAuthentication } from '../shared/framework/swagger/api.key.security';
 import { SdkGroupName } from '../shared/framework/swagger/sdk.decorators';
 
+/**
+ * @deprecated use controllers in /workflows directory
+ */
+
 @ApiCommonResponses()
```

Comment:
Remove extra line

```suggestion
 */
@ApiCommonResponses()
```

Reply:
are you sure about breaking? 
anyhow removign